kalite/distributed/templates/distributed/base.html

```diff
@@ -168,9 +168,10 @@
                           <ul class="nav navbar-nav navbar-right nav-tabs top-nav" role="tablist">
                               <div class="visible-xs">
                                   <li id="search_input">
-                                      <form class="navbar-form " action="{% url 'search' %}" method="get"  role="search">
+									  <form class="navbar-form " action="{% url 'search' %}" method="get"  role="search">
                                           {% comment %}translators: this will appear in the navigation bar. please try to keep it as short as possible.{% endcomment %}
-                                          <input type="text" name="query" id="search_box" class="form-control pull-left" placeholder="{% trans 'Topic, video, exercise...' %}" />
+                                          <p class="search-label search-label-mini"><label for="search_box">Search</label></p>
```

Comment:
Yes It affects the layout, But such measure should be taken to make it accessible well. I have also inspected using screen reader to look at how it affects users who use screen readers. I think this is a good solution.

![1](https://cloud.githubusercontent.com/assets/8178368/6672490/16b843ee-cbca-11e4-91e0-60e48f7dd1dc.png)

![2](https://cloud.githubusercontent.com/assets/8178368/6672491/171282a0-cbca-11e4-9932-c04440f010c8.png)